### ISSUE TITLE
Question on the published paper

### ISSUE THREAD

**Issue by nikhilh-20** [JavierYuste/AvaddonDecryptor#2]
Hello,

I had a question regarding function addresses mentioned in the paper. On page 9, there are two statements:

1. `The function responsible of generating the session key is the one located at 0x413f50`
2. `Finally, the function located at 0x413f50 is the one responsible for securely destroying the keys.`

On page 11, this statement: `However, we also hinted that the function responsible for destroying the cryptographic material was in fact never called if the system was not powered off.`

I don't understand. If function at `0x413f50` was used to generate the session key then it was surely called. Is the address of the function that is used to destroy the session keys mentioned wrong in the paper?

**Comment by JavierYuste** [comment]
Hello,

On page 8, in the second paragraph of Section 3.9, we indicated the correct address of the function responsible for generating the key, which is 0x413a60. However, on page 9, as you correctly indicate, we stated that it is located at 0x413f50. This is an error, since 0x413f50 is the address of the function responsible for destroying the keys. Therefore, it should be read as follows:

`The function responsible of generating the session key is the one located at 0x413a60`

Thank you for spotting that mistake, we will make sure to fix it.
